src/doplarr/backends/radarr.clj

```diff
@@ -19,16 +19,20 @@
     (let [quality-profiles (a/<! (impl/quality-profiles))
           rootfolders (a/<! (impl/rootfolders))
           {:keys [radarr/quality-profile]} @state/config
-          default-profile-id (utils/id-from-name quality-profiles quality-profile)]
+          default-profile-id (utils/id-from-name quality-profiles quality-profile)
+          default-root-folder (utils/id-from-name rootfolders rootfolders)]
       (when (and quality-profile (nil? default-profile-id))
         (warn "Default quality profile in config doesn't exist in backend, check spelling"))
+      (when (and rootfolders (nil? default-root-folder))
```

Comment:
Again, this should be a check against the singular rootfolder selection, not the set of rootfolders